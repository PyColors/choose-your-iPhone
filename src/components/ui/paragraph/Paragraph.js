import React, { Component } from 'react';
import PropTypes from 'prop-types';
//import './Paragraph.scss';

class Paragraph extends Component {

    render() {
        const { paragraph } = this.props;
    
        return (
          <p className="paragraph">
            {paragraph}cc
          </p>
        );
    }
}

Paragraph.propTypes = {
    paragraph: PropTypes.string.isRequired
};

export default Paragraph;
