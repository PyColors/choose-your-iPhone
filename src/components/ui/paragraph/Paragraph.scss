.paragraph {
    margin: 0 0 35px;
}
