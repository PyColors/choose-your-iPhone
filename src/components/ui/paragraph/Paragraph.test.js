import React from 'react';
import { shallow } from 'enzyme';
import Paragraph from './Paragraph';

test('Render a p', () => {
    const wrapper = shallow(
        <p>Hello Jest!</p>
    );
    expect(wrapper).toMatchSnapshot();
});
