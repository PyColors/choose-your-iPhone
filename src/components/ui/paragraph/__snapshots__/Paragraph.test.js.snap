// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a p 1`] = `
<p>
  Hello Jest!
</p>
`;
